### ISSUE TITLE
Namespace part of pageId is hardcoded

### ISSUE THREAD

**Issue by mikabytes** [tidenjs/tiden#25]
**Describe the bug**
when creating a new page using `tiden create page myPage myNamespace/otherNamespace` a new page is created. It works, but the id is wrong

```
const id = `one/two/myPage`
```

**Expected behavior**
The id should be:

```
const id = `myNamespace/otherNamespace/myPage`
```

**Comment by mikabytes** [comment]
@Zanshe Maybe you'd like to take a stab at this? The guilty files are:

```
ops/createPage.js
ops/createPage.test.js
```

Run the tests with `npm run test-ops`

**Comment by Zanshe** [comment]
Sure, I can take a look at it.

**Comment by mikabytes** [comment]
Fixed in #31
